Added new stations for Poland

Triloworld opened the pull request:
Local and official stations in Poland. Updated one station

Triloworld committed:
Added new stations

Local and official stations in Poland. Updated one station

themak1985 commented on channels/pl.m3u:
```diff
@@ -1,53 +1,79 @@
+http://hbbtvlive.v3.tvp.pl/hbbtvlive/livestream.php?app_id=tvpinfo
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="",TVP Parlament
+http://hbbtvlive3.v3.tvp.pl/hbbtvlive/livestream.php?app_id=kanal1
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="Polish" tvg-logo="" group-title="TVP3",TVP3 Białystok
```
group title needs to be blank or ideally one of the categories we use.

Triloworld replied:
Ok. I wasn't understand that feature. I will clear that text.

themak1985 commented on channels/pl.m3u:
```diff
@@ -1,53 +1,79 @@
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="https://i.imgur.com/hJAoLIq.png" group-title="",EZO TV
 http://live.ezotv.pl:1935/live/EZOTV/chunklist.m3u8
+#EXTINF:-1 tvg-id="" tvg-name="" tvg-language="" tvg-logo="" group-title="",Sfera TV
+http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/chunklist_w1591103044_b1128000_slpl.m3u8
```
If possible to create an adaptive link for this and echo24, would be preferred, but this is okay.

Triloworld replied:
Yes, I will fix that by using better one grabbed from website: 
http://stream.sferatv.pl:1935/sferalive/smil:sferalive.smil/playlist.m3u8

Triloworld replied:
Can you help? it don't work somehow ;/ It work with chunk one. I have fealing that chunk file won't work in future

themak1985 reviewed (commented):
Checked all the new links, they all work. Request to change a couple to adaptive links if possible, though not required. What is required through is removing the group-title you added with one of the categories we use or leave blank. tvg-id should also be blank unless there is a tv guide for poland.

Triloworld committed:
Updated list according to hints

Updated logo. Links fixed. Remove duplicated stream. Tested as they are working.

Triloworld commented:
Pleas check it once more as they won't be updated too soon.

Triloworld committed:
Update pl.m3u

Fixed one tv name mismatch as they are merged now

themak1985 commented:
> What ID correspond to? "unless there is a tv guide for poland." Probably exist but don't how I can spot It to fix that.
> Groups are "religion" for example? Is this "category"? I will re-check if that correct

Yes, group-id are the [categories](https://github.com/iptv-org/iptv#playlists-by-category) we have.  `group-id=""` is fine. 
Poland does not have a tv guide for our repo, so `tvg-id=""` and `tvg-name=""` is fine.
